src/renderers/dom/shared/__tests__/DOMPropertyOperations-test.js

```diff
@@ -261,4 +261,20 @@ describe('DOMPropertyOperations', () => {
       );
     });
   });
+
+  describe('getValueForProperty', function() {
+    it('uses getValueForAttribute when given a custom attribute', function() {
+      var node = document.createElement('div');
+
+      node.setAttribute('data-test', 'test');
+
+      var value = DOMPropertyOperations.getValueForProperty(
```

Comment:
I'm surprised why `aria` tests in integration test don't trigger this path. Can you check why?